stories/1-Button.stories.js

```diff
@@ -0,0 +1,49 @@
+import { action } from '@storybook/addon-actions';
```

Comment:
Tell me if we need to adapt ESlint configuration—or if we add a scoped one for stories maybe?

Reply:
Those were the type of problems that I fixed:

- `error  Filename is not in kebab case. Rename it to `alert.stories.js`
- `error Extra semicolon`
- `error Multiple spaces found before '='`

I don't think we need for the moment to adapt ESlint configuration.